ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+      | Egress       |                                                     100 |
+      | Storage      |                                                    5000 |
+      | Provider     | aws                                                     |
+      | Availability | LOW                                                     |
+      | Region       | us-west-2                                               |
+      | Status       | UP                                                      |
+      | Endpoint     | SASL_SSL://pkc-4kgmg.us-west-2.aws.confluent.cloud:9092 |
+      | ApiEndpoint  | https://pkac-ldgj1.us-west-2.aws.confluent.cloud        |
+      +--------------+---------------------------------------------------------+
+
+#. Specify ``lkc-x6m01`` as the active |ak|| cluster by running the following
+   command:
+
+   .. code-block:: bash
+
+      ccloud kafka cluster use lkc-x6m01
+
+   Running the previous command sets the |ak| cluster ``lkc-x6m01`` as the active
+   cluster for environment ``env-5qz2``.
+
+
+Create a user key/secret pair and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a user key/secret pair:
+
+   .. code-block:: bash
+
+      cloud api-key create --description "Demo credentials" --resource lkc-x6m01 -o json
+
+   The output is in JSON format:
+
+   .. code-block:: text
+
+      {
+         "key": "QX7X4VA4DFJTTOIA",
+         "secret": "fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D"
+      }
+
+#. Associate the API key ``QX7X4VA4DFJTTOIA`` to the Kafka cluster ``lkc-x6m01``:
+
+   .. code-block:: bash
+
+      ccloud api-key use QX7X4VA4DFJTTOIA --resource lkc-x6m01
+
+   Running the previoud command sets the API Key "QX7X4VA4DFJTTOIA" as the
+   active API key for ``lkc-x6m0``.
+
+#. Verify you see the following message:
+
+   .. code-block:: text
+
+      Waiting for Confluent Cloud cluster to be ready and for credentials to propagate
+      ....
+
+Produce and consume records with Confluent Cloud CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Produce records
+^^^^^^^^^^^^^^^
+
+#. Run the following command to create a new Kafka topic ``demo-topic-1``:
+
+   .. code-block:: bash
+
+      ccloud kafka topic create demo-topic-1
+
+#. Produce 10 messages to topic ``demo-topic-1`` by completing the following steps:
+
+   a. Implement the following logic:
+
+      .. code-block:: text
+
+         (for i in `seq 1 10`; do echo "${i}" ; done) | \ timeout 10s
+
+   b. Run the following command:
+
+      .. code-block:: bash
+
+         ccloud kafka topic produce demo-topic-1
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      Starting Kafka Producer. ^C or ^D to exit
+      1
+      2
+      3
+      4
+      5
+      6
+      7
+      8
+      9
+      10
+
+Consume records
+^^^^^^^^^^^^^^^
+
+#. Run the following command to consume messages from topic ``demo-topic-1``:
```

Comment:
Let's tell the user what the `-b` means.  (consume from beginning of topic)

Reply:
Updated